src/constants.rs

```diff
@@ -144,6 +144,7 @@ pub enum Chattype {
     Undefined = 0,
     Single = 100,
     Group = 120,
+    Mailinglist = 140,
```

Comment:
130 were verified groups, so better not reuse to protect against borked databases.